pkg/grpc/device_definition.proto

```diff
@@ -27,6 +27,18 @@ message GetDeviceDefinitionItemResponse {
 	repeated DeviceStyle device_styles = 9;
```

Comment:
let's mark `VehicleInfo vehicle_data = 5;` as deprecated in proto `VehicleInfo vehicle_data = 5 [deprecated = true];`.